card/mod/pointer/set/abstract/code_pointer.rb

```diff
@@ -3,7 +3,8 @@
 abstract_basket :item_codenames
 
 def content
-  item_codenames.map do |codename|
-    Card[codename] && Card[codename].name
-  end.compact.to_pointer_content
+  @cached_content ||=
```

Comment:
what clears this?

Reply:
`item_codenames` can only be changed with code but maybe you can break it if you change the names of the item cards